src/content/architecture/components/lighthouse/_index.en.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Lighthouse"
+date: 2020-03-02T21:25:35+01:00
+weight: 10
+---
+
+Lighthouse provides DNS discovery to Kubernetes clusters connected by [Submariner](https://github.com/submariner-io/submariner) in multi-cluster connected environments. The solution is compatible with any CNI (Container Network Interfaces) plugin.
+
+## Architecture
+The below digram shows the basic Lighthouse architecture.
+
+![Lighthouse Architecture](/images/lighthouse/architecture.png)
+
+### Lighthouse Controller
+This is the central discovery controller that gathers the information from the clusters, decides what information is to be shared and distributes the information as newly defined CRDs (Kubernetes custom resources).
+
+The Lighthouse controller will be running along with the [Admiral](https://github.com/submariner-io/admiral) control plane. It uses the API’s exposed by the Admiral to connect to each cluster and listen for Service CRUD. Once it is notified that a new Service is created and needs to be distributed, it creates the Lighthouse CRD and the same is distributed to all the clusters that are part of the control plane. The flow for update and delete will be similar.
```

Comment:
```suggestion
The Lighthouse controller uses the kubefed directory of clusters through the [Admiral](https://github.com/submariner-io/admiral) library. It connects to each cluster and listens for Service CRUD. Once it is notified that a new Service is created and needs to be distributed, it creates a MultiClusterService CR which is distributed to all the clusters that are part of the control plane. The flow for update and delete will be similar.
```

Reply:
Changes because there is no admiral service running side-by side, it's just an API

Reply:
I would also add that currently we depend on kubefed, and it's deployed along the lighthouse controller, although in the future we plan to break the kubefed dependency and make the lighthouse controller completely distributed.